template/admin/initServer.sqf

```diff
@@ -1,13 +1,81 @@
 // Do not Edit this file.
 
+// Defines player count for automated AAR.
+#define MIN_PLAYER_COUNT 5
+
+// Get weekday for start times
+tac_getWeekDay = {
+    params ["_date"];
+
+    // Keep only year, month, day if longer date format is given
+    _date = [_date select 0, _date select 1, _date select 2, 0, 0];
+
+    private _yearBefore = ((_date select 0) - 1) max 0;
+    private _leapYears = floor (_yearBefore / 4);
+    private _normalYears = _yearBefore - _leapYears;
+    private _days = _normalYears + (_leapYears * (366 / 365)) + dateToNumber _date;
+
+    (round (_days / (1 / 365))) mod 7  // return
+};
+
 // Mathematical Heresy
 systemTimeUTC params ["", "", "", "_hour", "_minute"];
-private _timeUntilStart = ((14 * 60) - (_hour * 60 + _minute)) * 60; // start time (1400z) - current time = time until start time
+private _weekDay = [systemTimeUTC] call tac_getWeekDay;
+private _startTime = [15, 14] select (_weekDay == 5); // 1400z on Saturday, 1500z otherwise
+private _timeUntilStart = ((_startTime * 60) - (_hour * 60 + _minute)) * 60; // start time - current time = time until start time
+_timeUntilStart = _timeUntilStart max 0;
+
+//diag_log format ["[TAC] Time Until Start: %1", _timeUntilStart];
```

Comment:
```suggestion
LOG_1("Time Until Start: %1",_timeUntilStart);
```
Check how this actually logs in RPT.

Reply:
"For Later" - Doesn't show in RPT at all.